index.html

```diff
@@ -36,65 +105,68 @@ <h2>
                 Main Program
             </h2>
             <img src="assets/images/guide-bar.svg" alt="guide bar image" id="guide-bar-mp">
-            <div class="program-list">
-                <div class="program-icon">
-                    <img src="assets/images/program-icon-1.png" alt="program icon 1">
-                </div>
-                <div class="program-title">
-                    Lecture
-                </div>
-                <div class="program-description">                    
-                    Listen to the speakers from various countries about the messeges of sharing and opening.
-                </div>
-            </div>
-            <div class="program-list">
-                <div class="program-icon">
-                    <img src="assets/images/program-icon-2.png" alt="program icon 2">
+            <div class="program-list-container">
+                <div class="program-list">
+                    <div class="program-icon">
+                        <img src="assets/images/program-icon-1.png" alt="program icon 1">
+                    </div>
+                    <div class="program-title">
+                        Lecture
+                    </div>
+                    <div class="program-description">                    
+                        Listen to the speakers from various countries about the messeges of sharing and opening.
+                    </div>
                 </div>
-                <div class="program-title">
-                    Exhibition
+                <div class="program-list">
+                    <div class="program-icon">
+                        <img src="assets/images/program-icon-2.png" alt="program icon 2">
+                    </div>
+                    <div class="program-title">
+                        Exhibition
+                    </div>
+                    <div class="program-description">                    
+                        Appreciate various creations applying CC license of artists, organized from Art Center
+                        Nabi.                
+                    </div>
                 </div>
```

Comment:
- Kindly, Program List is a section, and the program list are article, so let's update it change the `div` tags to `section` and each program list with an `article` tag to make it semantically correct 👍